pkg/core/native/policy.go

```diff
@@ -64,7 +62,7 @@ type Policy struct {
 	feePerByte              int64
 	maxBlockSystemFee       int64
 	maxVerificationGas      int64
-	blockedAccounts         BlockedAccounts
+	blockedAccounts         map[util.Uint160]bool
```

Comment:
I don't really like this change, this cache could still easily be `[]util.Uint160` with the new storage model, maps are a bit more heavy for this tasks. Though in practice it very well may be completely irrelevant.

Reply:
And it's simpler this way, of course.

Reply:
Replaced with array.